resources/keywords/deploy_application.resource

```diff
@@ -0,0 +1,98 @@
+    Set Test Variable    ${appname}
+    ${bindName}=    Update Service Binding    ${newProject}    ${instanceUUID}    ${appname}
+    Set Test Variable    ${bindName}
+    Create Service Binding      ${newProject}
+    Wait Until Keyword Succeeds    3x    2s     Wait Until Page Contains Element    ${elemSrvcBindArrow}
+    Click Element   ${elemSrvcBindArrow}
+    ${elemIconDeploymentUpd}=       Replace String    ${elemIconDeployment}    <<appname>>    ${appname}
+    Instance Pan Displays   ${bindName}
+
+User Selects Openshift Application For ${databaseProvider}
+    Element Should Be Visible    ${lnkTopology}
+    Click Element    ${lnkTopology}
+    ${appname}=     Get Import Application Name    ${databaseProvider}
+    Set Test Variable    ${appname}
+    ${elemIconDeploymentUpd}=       Replace String    ${elemIconDeployment}    <<appname>>    ${appname}
+    Wait Until Page Contains Element    ${elemIconDeploymentUpd}
+    Click Element    ${elemIconDeploymentUpd}
+    ${hdrDeploymentPanUpd}=       Replace String    ${hdrDeploymentPan}    <<appname>>    ${appname}
+    Wait Until Page Contains Element    ${hdrDeploymentPanUpd}
+    Wait Until Keyword Succeeds    5x    1s    Application Pod Loads With Running Status
+
+Application Pod Loads With ${given} Status
+    ${podstatus}=   Get Element Attribute    ${txtPodStatus}    innerText
+    Should Match    ${given}    ${podstatus}
+
+The Application Accesses The Connected ${database} Database Instance
+    Application Pod In Running Status
+    Access Data From Connected ${database} Instance
+
+Application Pod In Running Status
+     Close Topology Pan
+    ${txtApplicationUpd}=  Replace String    ${txtApplication}  <<appname>>    ${appname}
+    Click Element    ${txtApplicationUpd}
+    ${txtAppPodStatusUpd}=     Replace String    ${txtAppPodStatus}    <<appname>>    ${appname}
+    Wait Until Keyword Succeeds    10x    1s    Wait Until Page Contains Element    ${txtAppPodStatusUpd}
+
+Close Topology Pan
+    ${status}=  Run Keyword And Return Status    Element Should Be Visible    ${btnClosePan}
+    Run Keyword If    ${status}
+    ...    Click Element    ${btnClosePan}
+
+Access Data From Connected ${database} Instance
+    ${linkName}=    Catenate    SEPARATOR=-     ${appname}      ${newProject}
+    ${Application_URL}=     Get Quarkus Application URL   ${OCP_CONSOLE_URL}      ${linkName}
+    Execute Javascript      window.open("${Application_URL}")
+    Switch Window   url:${Application_URL}
+    Load Data To ${database} Database From Application
+
+Load Data To ${database} Database From Application
+    Run Keyword If    "Mongo" in """${database}"""
+    ...    Load Data To Mongo Application
+
+Load Data To Mongo Application
+    ${appStatus}=    Run Keyword And Return Status    Element Should Be Visible    ${hdrFruitApp}
+    Run Keyword If    not ${appStatus}
+    ...    Reload Page
+    Run Keyword And Return Status    Element Should Be Visible    ${hdrFruitApp}
```

Comment:
@mrrajan Could you please change this to `Wait Until Element Is Visible    ${txtBxFruitName}`